quickstarts/Embeddings.ipynb

```diff
@@ -35,18 +35,33 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 2,
       "metadata": {
-        "id": "YD6urJjWGVDf"
+        "id": "YD6urJjWGVDf",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "d910e365-81ef-4927-e8b5-b9f82de8d847"
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "  Installing build dependencies ... \u001b[?25l\u001b[?25hdone\n",
+            "  Getting requirements to build wheel ... \u001b[?25l\u001b[?25hdone\n",
+            "  Preparing metadata (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
+            "  Building wheel for google-generativeai (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n"
+          ]
+        }
+      ],
       "source": [
-        "!pip install -U -q google.generativeai # Install the Python SDK"
+        "!pip install -q \"git+https://github.com/google/generative-ai-python.git@main\""
```

Comment:
If you're including this please add a comment explaining that `output_dimensionalit` is not in 0.5.1 but will b e in the next release.